src/SettingsStructs.h

```diff
@@ -44,6 +44,8 @@ struct FixedPageUI {
     Vec<COLORREF>* gradientColors;
     // if true, TextColor and BackgroundColor will be temporarily swapped
     bool invertColors;
+    // if true, hides the scrollbars but retain ability to scroll
+    bool hideScrollbars;
 };

```

Comment:
See top of the file. The source of this is in `gen_settings_structs.go`. So add it there and then run `doit.bat -gen-structs` to regenerate this file. Requires go to be installed.